packages/playground/package.json

```diff
@@ -59,19 +59,21 @@
     "ajv-formats": "^2.1.1",
     "ajv-i18n": "^4.2.0",
     "antd": "^4.22.8",
+    "atob": "^2.1.2",
```

Comment:
Not really associated with this line, but you need to update the `lint-staged` block above to change to `ts?(x)` instead of `js`